prac_06/car.py

```diff
@@ -0,0 +1,36 @@
+"""CP1404/CP5632 Practical - Car class example."""
+
+
+class Car:
+    """Represent a Car object."""
+
+    def __init__(self, name="", fuel=0):
+        """Initialise a Car instance.
+
+        fuel: float, one unit of fuel drives one kilometre
+        name: string, cars name
+        """
+        self.name = name
+        self.fuel = fuel
+        self._odometer = 0
+
+    def __str__(self):
+        return f"{self.name}, fuel ={self.fuel}, odometer={self._odometer} "
```

Comment:
Tiny thing, but you have a space character between 'fuel' and the '=' assignment operator. When this runs it will print as "fuel =150, odometer=30" (or whatever values), i.e. inconsistent spacing.

Reply:
Thanks, fixed.